aten/src/ATen/native/mkldnn/xpu/detail/Utils.h

```diff
@@ -80,7 +82,7 @@ dnnl::memory::dims compatible_dilation(Vec&& dilation) {
     *it -= 1;
   }
   return ret;
-}
+};
```

Comment:
Why this change is needed?
```suggestion
}
```